SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -78,8 +78,9 @@ bool USpatialStatics::IsActorGroupOwnerForActor(const AActor* Actor)
 		return false;
 	}
 
+	// Offloading using the Unreal Load Balancing always load balances based on the owning actor.
 	const AActor* RootOwner = Actor;
-	while (RootOwner->bUseNetOwnerActorGroup && RootOwner->GetOwner() != nullptr)
+	while (RootOwner->GetOwner() != nullptr)
```

Comment:
To the first, yes we decided it could be removed entirely. I've updated the "Cleanup the engine" ticket to also include removing it.
To the second, I had been thinking that you would have an AI controller per layer, but if you think it makes more sense to have a replicated check here instead, I can do that. I would then add a similar check in the LayeredLB, because I think the most important thing is that they're consistent.

However, is there a chance that one class in the ownership heirarchy would be non-replicated but then the root is replicated? In that case, stopping if you see a non-rep class would break things.

Reply:
Discussed offline. Conclusion was to have the SpatialStatics and the LayeredLB code both go up the ownership hierarchy, but only if the owner is also replicated.